cmd/peer-rest-client.go

```diff
@@ -141,15 +128,7 @@ func (client *peerRESTClient) doNetOBDTest(ctx context.Context, dataSize int64,
 	buflimiter := make(chan struct{}, threadCount)
 	errChan := make(chan error, threadCount)
 
-	totalTransferred := int64(0)
-	transferChan := make(chan int64, threadCount)
-	defer close(transferChan)
-
-	go func() {
-		for v := range transferChan {
-			atomic.AddInt64(&totalTransferred, v)
-		}
-	}()
+	var totalTransferred int64
```

Comment:
Where is totalTransferred getting set now since previous block accumulating transferred bytes is gone?

Reply:
in the respective go-routines @poornas

Reply:
I just updated that.